releases/release-1.31/release-notes/maps/pr-126067-map.yaml

```
@@ -0,0 +1,9 @@
+pr: 126067
+releasenote:
+  text: |-
+    Graduated the Job SuccessPolicy to beta.
+
+    The new reason label, "SuccessPolicy" and "CompletionsReached" are added to the "jobs_finished_total" metric.
+    Additionally, If we enable the "JobSuccessPolicy" feature gate, the Job gets "CompletionsReached" reason for the "SuccessCriteriaMet" and "Complete" condition type
```

```suggestion
    Additionally, if you enable the `JobSuccessPolicy` feature gate, the Job gets "CompletionsReached" reason for the "SuccessCriteriaMet" and "Complete" condition type.
```

---

```suggestion
    Additionally, if you enable the `JobSuccessPolicy` feature gate, the Job gets "CompletionsReached" reason for the "SuccessCriteriaMet" and "Complete" condition type
```
@sftim the sentence continues so no need for full stop.

---

Noted